post_sign.py

```diff
@@ -0,0 +1,33 @@
+# Koji callback for writing signed rpms after signing via sign.py
+# Required for package generation with strict keys in mash
+
+from koji.plugin import register_callback
+import kojihub
+import logging
+from configparser import ConfigParser
+
+# Configuration file in /etc like for other plugins
+config_file = '/etc/koji-hub/plugins/sign.conf'
+
+
+def post_sign(cbType, sigkey=None, sighash=None, build=None, rpm=None):
+    """ Run the kojihub write-signed-rpm command on rpms after they've been signed with a signing key,
+    which is required for generating repos that use a specific key.
+    """
+    buildroot = kojihub.get_buildroot(rpm['buildroot_id'])
+    tag_name = buildroot['tag_name']
+    logging.getLogger('koji.plugin.sign').info("Running post-sign plugin for package with tag_name %s", tag_name)
```

Comment:
```suggestion
    logging.getLogger('koji.plugin.sign').info("Running post-sign plugin for build %s with tag_name %s", build, tag_name)
```

Reply:
also, you should log and skip writing the rpm if `sigkey` or `sighash` are `None`.

Reply:
or if `build` is `None`